lib/stripe/util.rb

```diff
@@ -54,6 +54,7 @@ def self.object_classes
         'order' => Order,
         'order_return' => OrderReturn,
         'three_d_secure' => ThreeDSecure,
+        'apple_pay_domain' => ApplePay::Domain,
```

Comment:
our API endpoint is /apple_pay/domains -- so I thought the sub-namespace might be more consistent? happy to change it to ApplePayDomain, considering nothing else had a sub-namespace